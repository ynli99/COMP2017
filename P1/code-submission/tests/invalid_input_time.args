Sydney Tokyo "09:00:00"
