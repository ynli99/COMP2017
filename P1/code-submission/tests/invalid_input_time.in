Sydney::Tokyo::09:60:59
Sydney::Tokyo::9:10:40
Sydney::Tokyo::10:50:70
Sydney::Tokyo::14:20:00
