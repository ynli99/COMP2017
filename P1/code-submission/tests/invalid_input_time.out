The next train to Tokyo from Sydney departs at 14:20:00
